jobserver/models/outputs.py

```diff
@@ -142,6 +143,19 @@ class Meta:
             )
         ]
 
+    def __format__(self, format_spec):
+        match format_spec:
+            case "b":
+                return f"{self.size}b"
```

Comment:
```suggestion
                return f"{self.size:,}b"
```

Large bytes are hard to read, so let's use comma separators?